### PR TITLE
Need parens to use bitwise and as function call

### PR DESCRIPTION
**Pull request by ethomag**
See: https://discourse.julialang.org/t/bitwise-and-does-not-parse-as-function-call/1274

### CODE REVIEW

File: base/docs/helpdb/Base.jl [hunk 1]
```diff
@@ -644,9 +644,9 @@ julia> exp10(0.2)
 exp10
 
 """
-    &(x, y)
+    x & y
 
-Bitwise and.
+Bitwise and. Add parenthesis for function application form: (&)(x, y)
```
Comment (tkelman) [inline base/docs/helpdb/Base.jl]: code highlight and remove the trailing whitespace

File: base/docs/helpdb/Base.jl [hunk 2]
```diff
@@ -644,9 +644,9 @@ julia> exp10(0.2)
 exp10
 
 """
-    &(x, y)
+    x & y
 
-Bitwise and.
+Bitwise and. Add parenthesis for function application form: `(&)(x, y)`
```
Comment (giordano) [inline base/docs/helpdb/Base.jl]: I'm not a native English speaker (so I may be wrong), but shouldn't it be "parentheses" (plural)?

**Comment by ethomag** [comment]
@giordano Thanks. I'm not a native english speaker either, but your suggestion sounds right to me.  So I changed the PR.

File: base/docs/helpdb/Base.jl [hunk 3]
```diff
@@ -644,9 +644,9 @@ julia> exp10(0.2)
 exp10
 
 """
-    &(x, y)
+    x & y
```
Comment (stevengj) [inline base/docs/helpdb/Base.jl]: Should we document all of the binary operators in this form?

Maybe, in the old RST format, we needed to put the function name first in order for the manual-generator to find it, but that shouldn't be necessary any more.

**Pull request closed without merging**